lib/node/http.js

```diff
@@ -161,6 +162,10 @@ class HTTP extends Server {
           adjusted: this.network.now(),
           offset: this.network.time.offset
         },
+        bindings: {
+          sha256: sha256.native === 2 ? 'native' : 'js',
+          secp256k1: secp256k1.native === 2 ? 'native' : 'js'
```

Comment:
Couldn't there also be the nodejs backend?

Reply:
> Couldn't there also be the nodejs backend?

Good point. I was thinking JavaScript is JavaScript when I wrote this. From the last time that I looked at the code, the main difference between the node backend and the js backend was the usage of `BigInt` in the node backend but browsers also natively support `BigInt` these days. Ideally, there would be an integer <--> string mapping of these values in `bcrypto`